packages/rocketchat-action-links/loadStylesheets.js

```diff
@@ -1,3 +1,3 @@
-RocketChat.theme.addPackageAsset(function() {
-	return Assets.getText('client/stylesheets/actionLinks.less');
+RocketChat.theme.addPackageAsset(() => {
```

Comment:
This function will be totally removed, please don't use this. If a package needs to set up a color, it should use a base class form the theme.